src/coauthor_interface/backend/api_server.py

```diff
@@ -224,41 +226,45 @@ def query():
 
     # Query GPT-3
     try:
-        if "---" in prompt:  # If the demarcation is there, then suggest an insertion
-            prompt, suffix = prompt.split("---")
-            response = openai.Completion.create(
-                engine=engine,
-                prompt=prompt,
-                suffix=suffix,
-                n=n,
-                max_tokens=max_tokens,
-                temperature=temperature,
-                top_p=top_p,
-                presence_penalty=presence_penalty,
-                frequency_penalty=frequency_penalty,
-                logprobs=10,
-                stop=stop_sequence,
-            )
+        if DEV_MODE:
+            # DEV_MODE: return no suggestions
+            suggestions = []
         else:
-            response = openai.Completion.create(
-                engine=engine,
-                prompt=prompt,
-                n=n,
-                max_tokens=max_tokens,
-                temperature=temperature,
-                top_p=top_p,
-                presence_penalty=presence_penalty,
-                frequency_penalty=frequency_penalty,
-                logprobs=10,
-                stop=stop_sequence,
-            )
-        suggestions = []
-        for choice in response["choices"]:
-            suggestion = parse_suggestion(
-                choice.text, results["after_prompt"], stop_rules
-            )
-            probability = parse_probability(choice.logprobs)
-            suggestions.append((suggestion, probability, engine))
+            if "---" in prompt:  # If the demarcation is there, then suggest an insertion
+                prompt, suffix = prompt.split("---")
```

Comment:
Limit the number of splits to 1 in case we get something like `aaa---bbb---ccc`.